_posts/2021-03-16-galaxy-21-01.md

```diff
@@ -0,0 +1,39 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+tags: [release]
+title: UseGalaxy.eu update to 21.01
+supporters:
+ - denbi
+ - elixir
+---
+
+UseGalaxy.eu has been updated to 21.01, a few days before the official release. Enjoy the preview
+
+
+## Some Highlights
+
+### New History Panel (Beta)
+  This release includes a preview of the new History Panel. The new history menu can be enabled and disabled in the options menu of the History Panel.
+
+### Performance Improvements
+  This release includes many performance improvements for browsing and searching Data libraries (Many thanks to `Oleg Zharkov <https://github.com/OlegZharkov>`__, `Pull Request 10204`_, `Pull Request 11099`_, `Pull Request 10557`_, `Pull Request 10558`_, `Pull Request 10807`_),
+  and improved job creation performance for certain classes of tools and copying Histories, Datasets and Dataset Collections (`Pull Request 10539`_, `Pull Request 10549`_).
+  The performance of reading tool wrappers when starting Galaxy has been much improved. We have therefore decided to disable the tool document cache by default,
+  since it only provides negligible speedups (`Pull Request 10804`_, `Pull Request 11447`_). The tool document cache can be re-enabled with the `enable_tool_document_cache <https://github.com/galaxyproject/galaxy/blob/283ec1772fcc0e574997dc2c5ad12dc86e3e1d37/lib/galaxy/config/sample/galaxy.yml.sample#L626>`_ setting.
+
+### Offload Zip Archive Creation to NGINX
```

Comment:
this is too technical I think for the users. Do we use this actually?

Reply:
I agree, we can remove it.